apps/demos/Demos/DataGrid/VirtualScrolling/description.md

```diff
@@ -1,6 +1,6 @@
-If the DataGrid component is bound to a large dataset, you can enable the virtual scroll feature to optimize data load times and improve user navigation. The component calculates the overall number of visible rows and displays a scrollbar that allows users to navigate to any section of rows. When users release the scroll thumb, the control loads records to be displayed in the viewport and removes other rows from memory. 
+To optimize data load times and improve user navigation when binding DataGrid to a large dataset, you can enable virtual scrolling. In this demo, we bind the component to a dataset of 100,000 records. 
 
-To allow users to scroll the DataGrid virtually, set the **scrolling**.[mode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#mode) to *"virtual"*.
+To enable virtual scrolling, set **scrolling**.[mode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#mode) to *"virtual"*.
 <!--split-->
 
-In this demo, the DataGrid is bound to a local dataset of 100,000 records. You can drag the scrollbar on the right to see that records within the viewport are updated immediately.
\ No newline at end of file
+Unlike [infinite scrolling](/Demos/WidgetsGallery/Demo/DataGrid/InfiniteScrolling/MaterialBlueLight/), virtual scrolling allows users to navigate to any section of rows immediately. DataGrid only loads displayed rows into memory and unloads rows as they are hidden by user scrolling. When virtual scrolling is enabled, **Ctrl + Home** and **Ctrl + End** navigate to the component's first/last rows.
```

Comment:
> When virtual scrolling is enabled, **Ctrl + Home** and **Ctrl + End** navigate to the component's first/last rows

More precisely, they navigate to the first cell of the first row/last cell of the last row.